MapboxNavigation/RouteTableViewController.swift

```diff
@@ -72,7 +72,17 @@ extension RouteTableViewController: UITableViewDelegate, UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
-        return "Leg \(section + 1)"
+        // Don't display section header if there is only one step
+        guard routeController.routeProgress.route.legs.count > 1 else {
+            return nil
+        }
+        
+        let legToFrom = routeController.routeProgress.route.legs[section].description.components(separatedBy: ", ")
```

Comment:
`RouteLeg.description` is equivalent to `RouteLeg.name`. `name` lists the main streets along the leg, not the leg’s origin and destination. For example, `name` might be “Market Street, I-10, Main Street”. If you want “Biloxi” and “Mobile”, you need to look at the `name` properties of RouteLeg’s `source` and `destination` properties, which won’t be very useful until mapbox/MapboxDirections.swift#135 is fixed.